### PR TITLE
journal-cms--prod full database post mortem

### PR DESCRIPTION
**Pull request by giorgiosironi**
Timeline reported from Slack channel and  New Relic emails, for later post-mortem.

Maybe we should move these to https://github.com/elifesciences/tech-team due to them being eLife-specific?

### CODE REVIEW

File: incidents/2018-01-13-Journal-cms-full-database.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+MTTR: 7h34m from alert to /ping working again
+
+## Corrective Actions
+
+- alert on RDS space [GIORGIO, LUKE]
+- check or update the TTL of journal's caches (API and CloudFront, IIIF). Can we turn off journal-cms for 72 hours and rely on caches? [CHRIS]
+- investigation: see if we can turn off revisions for article content (especially views) and clean up old data left there [NATHAN]
+- investigation: can we set up autoscaling on RDS space? [GIORGIO]
+- investigate: data gathering from New Relic to expose pages there as a dashboard [SEAN]
+- extract from logs where processes were hanging so that we can set up a timeout on the db connection [GIORGIO]
```
Comment (giorgiosironi) [inline incidents/2018-01-13-Journal-cms-full-database.md]: unclear from PHP log and Monolog logs why processes were hanging, the hypothesis is deadlock so connection timeouts wouldn't make much of a difference in this case

File: incidents/2018-01-13-Journal-cms-full-database.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- possibly move Drupal’s lock to Redis [CHRIS to clarify]
+- logging in recommendations may be insufficient to detect the problem [GIORGIO]
```
Comment (giorgiosironi) [inline incidents/2018-01-13-Journal-cms-full-database.md]: logging in `/srv/recommendations/var/logs/all.json` seems to work well, but `error.json` seems not to be written anymore.

**Comment by seanwiseman** [comment]
As a task given to me from the post-mortem, I have taken a look at New Relic's custom dashboard options with the view of possibly getting more of a visual warning of this kind of issue in the future (depending on the visibility of said dashboard) before its breaking point.

In short, they have a bunch of options for creating custom boards but the layouts they offer make it slightly more difficult to track multiple services across multiple metrics so we would have to be picky. 

The next step, if people want to continue down this road, is to list what we would like to track, on what services, and I can build a draft board. Alternatively, if people want to just dive straight in just go to `tools > create custom dashboard` and draft one (enjoy the clunky UI).

File: incidents/2018-01-13-Journal-cms-full-database.md (hunk 2 above)
Comment (giorgiosironi) [reply incidents/2018-01-13-Journal-cms-full-database.md]: working on https://github.com/elifesciences/recommendations/pull/110

**Comment by giorgiosironi** [comment]
Merging to keep all the Markdown files together.

**Pull request merged**